benchopt/runner.py

```diff
@@ -431,7 +431,7 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
                                 f"{tag} {colorify('skip', YELLOW)}"
                             )
                             if reason is not None:
-                                print(f'Reason: {reason}')
+                                print(f'  Reason: {reason}')
```

Comment:
```suggestion
                                print(f'    Reason: {reason}')
```

Reply:
I think this should have 4 spaces to align with the solver.

```bash
Simulated[n_samples=200,n_features=500]                                                                                                               
  |--L2 Logistic Regression[fit_intercept=False,lmbd=1.0]                                                                                             
    |--sklearn[liblinear]: done (not enough run)                                                                                                      
    |--sklearn[newton-cg]: done (not enough run)                                                                                                      
    |--sklearn[lbfgs]: done (not enough run)                                                                                                          
    |--sklearn[sag]: done (not enough run)                                                                                                            
    |--sklearn[saga]: done (not enough run)                                                                                                           
    |--sklearn[sgd]: skip                                                                                                                             
  Reason: Not enough samples for SGD
  |--L2 Logistic Regression[fit_intercept=False,lmbd=0.01]                                                                                            
    |--sklearn[liblinear]: done (not enough run)                                                                                                      
    |--sklearn[newton-cg]: done (not enough run)                                                                                                      
    |--sklearn[lbfgs]: done (not enough run)                                                                                                          
    |--sklearn[sag]: done (not enough run)                                                                                                            
    |--sklearn[saga]: done (not enough run)                                                                                                           
    |--sklearn[sgd]: skip                                                                                                                             
  Reason: Not enough samples for SGD
Simulated[n_samples=1000,n_features=10]
```